server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/measure/Measure.java

```diff
@@ -133,11 +151,20 @@ public NewMeasureBuilder setQualityGateStatus(QualityGateStatus qualityGateStatu
       return this;
     }
 
+    /**
+     * @deprecated as only one period is now available. Use {@link #hasVariation()} instead
+     */
+    @Deprecated
     public NewMeasureBuilder setVariations(MeasureVariations variations) {
       this.variations = requireNonNull(variations, "Variations can not be set to null");
       return this;
     }
 
+    public NewMeasureBuilder setVariation(double variation) {
+      this.variations = createLeakVariation(variation);
+      return this;
+    }
+
     public Measure create(boolean value, @Nullable String data) {
       return new Measure(ValueType.BOOLEAN, developer, value ? 1.0d : 0.0d, data, null, description, qualityGateStatus, variations);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "Measure"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)